pkg/cmd/cli/cmd/deploy.go

```diff
@@ -227,7 +228,9 @@ func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer)
 	config.Status.LatestVersion++
 	_, err = o.osClient.DeploymentConfigs(config.Namespace).Update(config)
 	if err == nil {
-		fmt.Fprintf(out, "Started deployment #%d\n", config.Status.LatestVersion)
+		fmt.Fprintf(out, "--> Started deployment #%d\n", config.Status.LatestVersion)
+		fmt.Fprintf(out, "    Use '%s logs -f dc/%s' to track its progress.\n", o.baseCommandName, config.Name)
+		fmt.Fprintf(out, "    Use '%s scale --replicas=N dc/%s' to scale it accordingly.\n", o.baseCommandName, config.Name)
```

Comment:
I am reluctant to suggest `oc scale` after starting a deployment but I wanted to discuss it. It would be a nice fit if the deployer scaled based on d.spec.replicas and not a fixed annotation. Currently, during a running deployment, the dc will be scaled up but the size will be persisted in the underlying deployment in the next sync interval. Proportional scaling would also be a nice-to-have. @ironcladlou thoughts?